specs/lifespan.rst

```diff
@@ -56,6 +59,38 @@ lifespan protocol. If you want to log an error that occurs during lifespan
 startup and prevent the server from starting, then send back
 ``lifespan.startup.failed`` instead.
 
+The ``extensions["lifespan.state"]`` dict is an empty namespace.
+Applications can store arbitrary data in this namespace.
```

Comment:
This line is unclear... In the next section it's specified to store in `lifespan["state"]`, but here it mentions that applications can use `extension["lifespan.state"]`... :thinking:

Was this supposed to be removed?

Reply:
Yes I think so